dist/main.js

```diff
@@ -16,7 +16,7 @@
 
 /***/ }),
 
```

Comment:
- [ ] As required for the project kindly make sure your project has the following features as no one of them is working:
  - A functionality for adding a new task (add a new element to the array) is implemented.
  - A functionality for editing task descriptions is implemented.
  - A function for deleting a task (remove an element from the array) is implemented.
  - By default, new tasks should have the property completed set to `false` and the property `index` set to the value of the new array length (i.e. if a 5th task is added to the list, the index of that task should equal to 5).
  - Deleting a task should update all remaining items' indexes, so they represent the current list order and are unique.
  - All changes to the To-Do List should be saved in local storage i.e when editing, deleting and adding a task.
**Here is what your project looks like:**
![todo](https://user-images.githubusercontent.com/83042742/184213260-4e76d70c-0d16-495a-957d-7b2b7c3b4aec.gif)

Reply:
@tresorsawasawa please be specific about what's working and what not . Local storage is working on my side.

Reply:
Hi @rebel216 
Right now the user **can not add a task** and as a task cannot be added I can not know if the local storage, the Editing functionality, and the deleting functionality are also working.

Reply:
@tresorsawasawa  please check on ur side .. everything is working on my side. 👍

Reply:
@rebel216 give again some munites to check again

Reply:
I can see some JS error in  console of browser(I the image u shared) .. it's not present on my side.Please check that. 👍

Reply:
Hi @rebel216 

I just checked again, and now it is working except for one issue: **as mentioned above the property **index** should be set to the value of the new array length (i.e. if a 5th task is added to the list, the `index` of that task should equal to `5`). On your project, the first `index` is **0** instead of `1`
**

Reply:
Done 👍